compiler/src/iree/compiler/Codegen/Dialect/PCF/Transforms/Passes.td

```diff
@@ -135,4 +135,23 @@ def ConvertSRefToMemRefPass : Pass<"iree-pcf-convert-sref-to-memref", ""> {
                            "::mlir::vector::VectorDialect"];
 }
 
+def ResolveTokensPass : Pass<"iree-pcf-resolve-tokens", ""> {
+  let summary = "Resolves synchronization scopes on `pcf.sref` types.";
+  let description = [{
+    Resolves synchronization scopes attached to `pcf.sref` types by expanding
+    them to their concrete representations. This pass runs before
```

Comment:
nit/pedantic: The pass doesn't really know about other passes, so I'd say that it **should run** before sref-to-memref.